.github/workflows/rules.yml

```diff
@@ -57,7 +59,7 @@ jobs:
     - name: Test with pytest core
       run: |
         pytest
```

Comment:
In case you still want (I'd suggest at a later time) to have the run also with `numba` enabled, we can explicitly add a:
```
NUMBA_DISABLE_JIT=0 pytest -m numba
```
since the `D:` in the configs prevents overriding explicitly defined env vars, and we could make a custom marker `numba` to select those tests that should be run w/ and w/o Numba (if only w/ we can mark as skip by default).

Reply:
Yes, that sounds interesting, but I would keep it for a next round.